sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -111,6 +115,41 @@ object RapidsPluginUtils extends Logging {
     }
   }
 
+  private def detectMultipleJar(propName: String, jarName: String, propsName: Seq[String], 
+      conf: RapidsConf): Unit = {
+    val classloader = ShimLoader.getShimClassLoader()
+    val possibleRapidsJarURLs = classloader.getResources(propName).asScala.toSet
+    val (rapidsJarsURLs, rapidsJarsVers) = possibleRapidsJarURLs.flatMap { 
+      url => {
+        val ver = scala.io.Source.fromInputStream(url.openStream()).mkString("") 
+        if (propsName.exists(ver.contains(_))) {
+          Some((url, ver))
+        } else {
+          None
+        }
+      }
+    }.unzip
+    lazy val rapidsJarsMsg = rapidsJarsURLs.toList.map(_.toString.split("!").head).mkString(",")
+    lazy val rapidsJarsVersMsg = rapidsJarsVers.toList.mkString(",")
```

Comment:
NIT: Personally the version message is not good enough, we can try `mkString("\n")` instead.

Reply:
Thanks, I rewrote the error message and used some `\n` as sep.